packages/schema/exported-schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "10.1.0",
+  "version": "10.1.1",
```

Comment:
This might be a minor version bump, not a patch, since you're adding stuff.  Pretty minor thing, not sure how serious we are about semver around here.

Reply:
@jasonkotenko good call. I'm curious if there's a non-prod semver (beta) I could park this as well.

Reply:
because of the way we make some assumptions in the code base, the version can only be `number.number.number`, unfortunately.